packages/scandipwa/src/component/ProductDownloadableLinks/ProductDownloadableLinks.component.js

```diff
@@ -98,15 +75,34 @@ export class ProductDownloadableLinks extends PureComponent {
         );
     }
 
-    renderLink = (link) => {
+    renderLink(link) {
+        const { isOpenInNewTab } = this.props;
+        const { sample_url } = link;
+
+        if (!sample_url) {
+            return null;
+        }
+
+        return (
+            <Link
+              to={ sample_url }
+              isOpenInNewTab={ isOpenInNewTab }
+              block="ProductDownloadableLink"
+              elem="SampleLink"
+            >
+                { __('Sample') }
+            </Link>
+        );
+    }
+
+    renderDownloadableLink = (link) => {
```

Comment:
Please don't use arrow functions as class methods.
You cannot call the parent method when you are extending such functions.